pkg/skyenv/values_darwin.go

```diff
@@ -22,3 +19,13 @@ func PackageConfig() PkgConfig {
 	pkgconfig.Hypervisor.EnableAuth = true
 	return pkgconfig
 }
+
+// UserConfig contains installation paths (for mac)
+func UserConfig() PkgConfig {
+	var usrconfig PkgConfig
+	usrconfig.Launcher.BinPath = "/Library/Application Support/Skywire/apps"
+	usrconfig.LocalPath = HomePath() + "/.skywire/local"
+	usrconfig.Hypervisor.DbPath = HomePath() + "/.skywire/users.db" //permissions errors if the process is not run as root.
```

Comment:
What are you trying to say with the comment? It seems it was copied from previous implementation for `opt`. Does it indeed fail if not run as root in HOME?

Reply:
accidental inclusion, and the original comment issue is now basically resolved